v2/emailpassword/testing/testing-with-postman.mdx

```diff
@@ -21,11 +21,18 @@ The following guide will go over the process of testing backend APIs with Postma
 - Postman does cookie management on its own. So you don't need to manually set cookies on each request.
 :::
 
+:::note
+We recommend using cookie-based sessions while testing with Postman, to make use of the built-in cookie manager.
+
+You can find more information about header-based sessions [here](../common-customizations/sessions/token-transfer-method).
+:::
```

Comment:
```suggestion
:::note
We recommend using cookie-based sessions while testing with Postman, to make use of the built-in cookie manager. This can be enabled by adding the `"st-auth-mode: cookie"` request header during the sign up API call as shown below.

You do not need to do this in your frontend code since our SDK does this for you. 
:::
```